--- prewikka-5.2.0/prewikka/views/datasearch/idmefv2.py	2024-01-26 09:54:04.257333123 +0000
+++ prewikka-5.2.0/prewikka/views/datasearch/idmefv2.py2	2024-02-03 01:50:48.809427089 +0000
@@ -118,6 +118,7 @@
         ("idmefv2.description", N_("Description")),
         ("idmefv2.source(0).ip", N_("Source")),
         ("idmefv2.target(0).ip", N_("Target")),
+        ("idmefv2.target(0).location", N_("Target location")),
     ])
 
     def _get_default_cells(self, obj, search):
