--- prewikka-5.2.0/scripts/prewikka-cli     2023-03-12 09:59:42.000000000 +0000
+++ prewikka-5.2.0/scripts/prewikka-cli2    2023-03-12 10:07:14.370345248 +0000
@@ -262,10 +262,6 @@
     interpreter = PrewikkaCLI()
     env.request = FakeRequest()

-    if getpass.getuser() != "prewikka":
-        print("*** %s" % _("prewikka-cli must be executed as the prewikka user"))
-        sys.exit(SYSTEM_ERROR)
-
     main.Core.from_config(args.config, ignore_errors=False)
     env.request.user = env.auth.authenticate(args.user, no_password_check=True)
     env.request.user.set_locale()
